great_expectations/expectations/expectation.py

```diff
@@ -297,6 +297,10 @@ class Config:
     expectation_type: ClassVar[str]
     examples: ClassVar[List[dict]] = []
 
+    _save_callback: Union[Callable[[Expectation], Expectation]] = pydantic.PrivateAttr(
```

Comment:
This type looks wrong since the default value is `None` and the types not `Optional`. Maybe some pydantic magic is confusing mypy (the real object is wrapped in a pydantic one). It's also a Union with currently only 1 element.  We do null checks below. Could you add `None` to the Union?

Reply:
thanks!